packages/appstore/src/components/modals/real-wallets-upgrade/real-wallets-upgrade.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import { useWalletMigration } from '@deriv/hooks';
+import { useWalletMigration } from '@deriv/api';
```

Comment:
We shouldn't be using hooks from api package here.

Reply:
Using hooks package now